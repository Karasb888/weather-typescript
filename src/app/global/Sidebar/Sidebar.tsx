import * as React from 'react';
import { Box } from '@material-ui/core';
import FavoriteCitiesList from '../FavoriteCitiesList/FavoriteCitiesList';

const Sidebar: React.FC = () => {
    return (
        <Box>
            <FavoriteCitiesList />
        </Box>
    );
};

export default Sidebar;
