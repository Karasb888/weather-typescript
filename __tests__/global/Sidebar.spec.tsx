import * as React from 'react';
import { shallow } from 'enzyme';
import Sidebar from '../../src/app/global/Sidebar/Sidebar';

describe('++++ <Sidebar /> ++++', () => {
    it('Renders correctly', () => {
        const wrapper = shallow(<Sidebar />);
        expect(wrapper).toMatchSnapshot();
    });
});
