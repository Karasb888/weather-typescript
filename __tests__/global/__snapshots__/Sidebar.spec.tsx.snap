// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`++++ <Sidebar /> ++++ Renders correctly 1`] = `
<Styled(MuiBox)>
  <Connect(FavoriteCitiesList) />
</Styled(MuiBox)>
`;
